src/OSPSuite.Core/Domain/Services/ValueComparer.cs

```diff
@@ -30,32 +22,50 @@ public static bool AreValuesEqual(double value1, double value2, double relTol)
       }
 
       /// <summary>
-      ///   returns true if the values of the given parameters are equals otherwise false
+      ///    returns <c>true</c> if the values of the given parameters are equals otherwise <c>false</c>
       /// </summary>
       public static bool AreValuesEqual(IParameter parameter1, IParameter parameter2)
       {
+         if (parameter1 == null && parameter2 == null)
```

Comment:
Added this checks that has been bugging me for a while in PKSim because I have to check that parameters are not null